variables.tf

```diff
@@ -27,25 +27,25 @@ variable "tags" {
   type        = map(string)
 }
 
-variable "enable_bucket_inventory" {
+variable enable_bucket_inventory {
```

Comment:
There's a mix of quoted and unquoted variables in variables.tf. Can we just continue using quoted?

Reply:
I will change.

Reply:
It's done, but why do you prefer quoted?

Reply:
I don't really have a preference, just want to maintain consistency